tests/functional/validate/08-whitespace/flow.cylc

```diff
@@ -0,0 +1,19 @@
+#!jinja2
+
+{% set HELLO=False %}
+[  scheduling   ]
+    initial cycle point = 20140101T00
+    [[  special tasks  ]]
+        sequential = foo , bar ,  baz
+        include at start-up =   \
+            foo {% if HELLO %}   ,    \
+            bar
+            {% endif %}
+    [[   graph  ]]
+        T00, T12  ="""
+        a => b
+        foo => bar &  baz  """
+
+   [  runtime   ]
+    [[  foo,bar   ,  a,  b  ]]
+%include "inc.cylc"
```

Comment:
Can't believe I'm saying this but there is some trailing whitespace *missing* from this file!

Reply:
Is there some way of preventing whitespace tests passing if the whitespace gets stripped by mistake?

Reply:
Could potentially write out the `flow.cylc` file then run `sed -i 's/$/ /'` on it to add one whitespace char to the end of each line.